webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+
+module.exports = {
+
+    mode: 'development',
+
+    entry: './way_to_home/static/index.js',
+
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'bundle.js',
+    },
+
+    watch: true,
+
+    watchOptions: {
+        aggregateTime: 100
+    },
+
+    module: {
+        rules: [
+              { test: /\.css$/, use: 'css-loader' },
+              { test: /\.ts$/, use: 'ts-loader' }
```

Comment:
it is not necessary! because we will not use TypeScript!

Reply:
deleted